src/v2/Artsy/Router/errors.tsx

```diff
@@ -0,0 +1,8 @@
+export class PermanentRedirectException extends Error {
```

Comment:
Maybe rename to `Exceptions` or even `PermanentRedirectException`? Just wondering if this is discoverable under `errors`. But i'm fine with it either way 👍